index.html

```diff
@@ -1,176 +1,780 @@
+                </div>
+            </form>
+        </div>
+    </section>
+</div>
 
-                }
+  <div class="main-banner wow fadeIn" id="top" data-wow-duration="1s" data-wow-delay="0.5s">
+    <div class="container" style="margin-bottom: 140px;">
+      <div class="row">
+        <div class="col-lg-12">
+          <div class="row">
+            <div class="col-lg-6 align-self-center">
+              <div class="left-content show-up header-text wow fadeInLeft" data-wow-duration="1s" data-wow-delay="1s">
+                <div class="row">
+                  <div class="col-lg-12">
+                    <h2>Login to foodprint</h2>
+                    <p>The recipe and diet planning app with full nutritional and ecological data. Available on Android, iOS and in browser! Register now and eat well with ease. Free forever, we promise!</p>
+                  </div>
+                  <!--
+                  <div class="col-lg-12">
+                    <div class="white-button first-button scroll-to-section">
+                      <a href="https://apps.apple.com/lt/app/cooking-fever/id714796093" target="_blank">App store <i class="fab fa-apple"></i></a>
+                    </div>
+                    <div class="white-button scroll-to-section">
+                      <a href="https://play.google.com/store/apps/details?id=com.nordcurrent.canteenhd" target="_blank"> Google Play<i class="fab fa-google-play"></i></a>
+                    </div>
+                  <br>
+                  -->
+                    <div class="white-button scroll-to-section">
+                      <a href="https://staging.quanteco.org/search" target="_blank"> Login Online</a>
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <div class="col-lg-6">
+                <div class="right-image wow fadeInRight" data-wow-duration="1s" data-wow-delay="0.5s">
+                  <img src="assets/images/qe/phone2.png" alt="" style="z-index: 3; width: 410px;">
+                </div>
+              </div>
+            </div>
+            <br>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div id="services" class="services section">
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-8 offset-lg-2">
+          <div class="section-heading  wow fadeInDown" data-wow-duration="1s" data-wow-delay="0.5s">
+            <h4>Eat with your heart</h4>
+              
+            <img src="assets/images/heading-line-dec.png" alt="">
+            <p> Choose from our growing library of recipes. See their environmental impact and nutritional values. Track your nutrition and CO<sub>2</sub> output on a daily basis. Together we can meet our climate goals.
+            </div>
+        </div>
+      </div>
+    </div>
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-3">
+          <div class="service-item first-service">
```

Comment:
The leaf that appears in all 4 cards makes the text hard to read. Maybe make it a lot softer so the text can remain black?